### PR TITLE
Allow changing scale factor for workspace

### PR DESCRIPTION
**Pull request by sandeshvijayraj**
## Summary

Current UI doesn't allow user to change scale factor for workspace. The MR allows user to modify scale factor from configuration page. The MR Also address small typo

## Test Plan

The MR changes can be tested locally.

## Changes

1. Pull the MR and from terminal go to web folder using command `cd ./web`
2. In case you are using RTDM project for the first time install dependencies using `yarn install`
3. Start the service using `yarn dev`
4. Visit `http://localhost:3000/configuration`

You can see in connection section you can also modify scaleFactor along with host details and workspace credentials

## Subscribers

@mivasconcelos @TitoGrine

### CODE REVIEW

File: web/src/data/constants.ts [hunk 1]
```diff
@@ -34,8 +34,8 @@ export const selectableCitiesData: Array<City> = [
   {
     id: 10998,
     name: "Paris",
-    centerLat: 43.19999996623008,
-    centerLon: -80.38332998797264,
+    centerLat: 2.34879996,
+    centerLon: 48.85340997,
     diameter: 0.4,
   },
```
Comment (TitoGrine) [inline web/src/data/constants.ts:40]: Can you confirm the `London` coordinates are also correct? I remember Arnaud also mentioning London was in the US and not the UK. Please check.

File: web/src/data/recoil.ts [hunk 2]
```diff
@@ -64,6 +64,7 @@ export const selectedCity = atom({
 export const selectedCities = atom<Array<City>>({
   key: "selectedCities",
   default: [],
+  // effects: [localStorageEffect()]
```
Comment (mivasconcelos) [inline web/src/data/recoil.ts:67]: was this commented by mistake?

Comment (sandeshvijayraj) [reply web/src/data/recoil.ts:67]: ya this line itself is not needed will be removed in next push

File: web/src/pages/Dashboard.tsx [hunk 3]
```diff
@@ -255,7 +255,6 @@ export const NotificationsMap = () => {
           selectionDropdownLeft={isSmallScreen ? undefined : "31.5%"}
           selectionDropdownTop={isSmallScreen ? undefined : "1vw"}
           useRenderer={useNotificationsRenderer}
-          zoom={13}
```
Comment (mivasconcelos) [inline web/src/pages/Dashboard.tsx:258]: any reason in particular to remove this?

**Review by mivasconcelos — APPROVED** [review]

File: web/src/pages/Dashboard.tsx (hunk 3 above)
Comment (sandeshvijayraj) [reply web/src/pages/Dashboard.tsx:258]: Yes I wanted to set it to 12 which is default value and no need to pass it separately

The reason is city name is visible on map if not zoomed >12 so set it to 12

**Pull request merged**